src/librustc_mir/interpret/place.rs

```diff
@@ -1136,12 +1135,13 @@ where
 
     pub fn raw_const_to_mplace(
         &self,
-        raw: RawConst<'tcx>,
+        alloc_id: AllocId,
+        layout: TyLayout<'tcx>,
     ) -> InterpResult<'tcx, MPlaceTy<'tcx, M::PointerTag>> {
```

Comment:
Now the function name has nothing to do with the signature any more... and I have no idea what the function is doing. At the very least, it needs a doc comment.